axum/src/extract/multipart.rs

```diff
@@ -224,7 +227,6 @@ composite_rejection! {
     ///
     /// Contains one variant for each way the [`Multipart`] extractor can fail.
     pub enum MultipartRejection {
-        BodyAlreadyExtracted,
         InvalidBoundary,
     }
```

Comment:
Do you want this to remain an enum for forwards compatibility?

Reply:
Nah I'd rather just remove it rather than not using it.

Reply:
I'm not sure you understood my question 😅

I wanted to ask whether you want to keep `MultipartRejection` even if it only has one variant now, or remove it in favor of using the inner type directly.

Reply:
Ah sorry. I thought you meant `BodyAlreadyExtracted`. Kinda wanna keep the enum especially since multipart interacts with an external dependency. So like if multer adds a new error type we might need a new variant here.